djhtml/modes.py

```diff
@@ -565,6 +574,8 @@ def create_token(self, raw_token, src, line):
                     Token.Open(raw_token, mode=DjHTML),
                     self.return_mode,
                 )
+            if not self.absolute:
```

Comment:
This particular case is indeed problematic without `token.dedents`. Solveable, but problematic.

Reply:
I don't mind "cheating" with tokens this way. It's actually quite elegant, and if we did it everywhere we wouldn't need the separate Open, Close, and CloseAndOpen classes at all. So I'll leave it in just to remind my future self that this is also a possibility.

Thanks for reviewing!